src/include/pagestat_inline.h

```diff
@@ -23,3 +23,61 @@
 
 /* Copy the values from one time page stat structure to another. */
 #define WT_PAGE_STAT_COPY(dest, source) (*(dest) = *(source))
+
+/* Merge a page stat structure into another - summing the value from each. */
+#define WT_PAGE_STAT_MERGE(ps, merge)                                                \
+    do {                                                                             \
+        /* Merge byte count */                                                       \
+        if ((ps)->byte_count == WT_STAT_NONE || (merge)->byte_count == WT_STAT_NONE) \
+            (ps)->byte_count = WT_STAT_NONE;                                         \
```

Comment:
Why we don't need to set the flag WT_RESET_ROW_COUNT

Reply:
We should probably set that flag but I'll combine the macros instead.